Externals_CAM.cfg

```diff
@@ -51,32 +51,47 @@ required = True
 
 [atmos_phys]
 protocol = git
-tag = atmos_phys0_00_011
+tag = atmos_phys0_02_006
 repo_url = https://github.com/NCAR/atmospheric_physics
 local_path = src/atmos_phys
 required = True
 
 [camnor_phys]
 protocol = git
-tag = 2aad4ca
-repo_url = https://github.com/NorESMhub/CAM-Nor-physics
+branch = gold2718_CAM-Nor-physics/noresm
+repo_url = https://github.com/mvertens/CAM-Nor-physics
 local_path = src/physics/camnor_phys
 required = True
 
 [oslo_aero]
 protocol = git
-tag = noresm_oslo_aero_v1
-repo_url = https://github.com/NorESMhub/OSLO_AERO
+#branch = feature/UpdateNoresmBranchRebase
+tag = 7abf42a
+repo_url = https://github.com/mvertens/OSLO_AERO
```

Comment:
This needs to be updated before merging.

Reply:
Should this point to a OSLO_AERO and CAM-NOR tags or hashes?